test/unit_tests/js/organisms/Footer-spec.js

```diff
@@ -0,0 +1,22 @@
+const BASE_JS_PATH = '../../../../cfgov/unprocessed/js/';
```

Comment:
Is it possible for these paths to be root-relative so that we could set `BASE_JS_PATH` once for the entire project?

Reply:
I haven't gotten a chance to deeply look into it, but we could probably use a solution in here https://gist.github.com/branneman/8048520. Let me know if any of those strike you as better than another.